website/docs/docs/cloud/dbt-cloud-ide/git-commit-signing.md

```diff
@@ -0,0 +1,80 @@
+---
+
+# Git commit signing <Lifecycle status="Enterprise" />
+
+To prevent impersonation and enhance security, you can sign your Git commits before pushing them to your repository. Using your signature, a Git provider can cryptographically verify a commit and mark it as "verified", providing increased confidence about its origin.
+
+You can configure dbt Cloud to sign your Git commits when using the IDE for development. To set up, enable the feature in dbt Cloud, follow the flow to generate a keypair, and upload the public key to your Git provider to use for signature verification.  
+
+
+## Prerequisites 
+
+- GitHub or GitLab is your Git provider. Currently, Azure DevOps is not supported.
+- You have a dbt Cloud account on the [Enterprise plan](https://www.getdbt.com/pricing/).
+
+## Generate GPG keypair in dbt Cloud
+
+To generate a GPG keypair in dbt Cloud, follow these steps:
+1. Go to your **Personal profile** page in dbt Cloud.
+2. Navigate to **Signed Commits** section.
+3. Enable the **Sign commits originating from this user** toggle.
+4. This will generate a GPG keypair. The private key will be used to sign all future Git commits. The public key will be displayed, allowing you to upload it to your Git provider.
+
+<Lightbox src="/img/docs/dbt-cloud/example-git-signed-commits-setting.png" width="95%" title="Example of profile setting Signed commits" />
+
+## Upload public key to Git provider 
+
+To upload the public key to your Git provider, follow the detailed documentation provided by the supported Git provider:
+
+- [GitHub instructions](https://docs.github.com/en/authentication/managing-commit-signature-verification/adding-a-gpg-key-to-your-github-account) 
+- [GitLab instructions](https://docs.gitlab.com/ee/user/project/repository/signed_commits/gpg.html) 
+
+Once you have uploaded the public key to your Git provider, your Git commits will be marked as "Verified" after you push the changes to the repository.
+
+<Lightbox src="/img/docs/dbt-cloud/git-sign-verified.jpg" width="95%" title="Example of a verified Git commit in a Git provider." />
+
+## Considerations
+
+- The GPG keypair is tied to the user, not a specific account. There is a 1:1 relationship between the user and keypair. The same key will be used for signing commits on any accounts the user is a member of.
+- The GPG keypair generated in dbt Cloud is linked to the email address associated with your account at the time of keypair creation. This email identifies the author of signed commits.
+- For your Git commits to be marked as "verified", your dbt Cloud email address must be a verified email address with your Git provider. The Git provider (such as, GitHub, GitLab) checks that the commit's signed email matches a verified email in your Git provider account. If they don’t match, the commit won't be marked as "verified."
+- Keep your dbt Cloud email and Git provider's verified email in sync to avoid verification issues. If you change your email address:
```

Comment:
```suggestion
- Keep your dbt Cloud email and Git provider's verified email in sync to avoid verification issues. If you change your dbt Cloud email address:
```